.github/scripts/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class BranchModifiedError(ValueError):
+    pass
+
+
+class KRSMaintenanceError(Exception):
+    pass
```

Comment:
Trochę zależy od przyjętej konwencji, ale skoro ten błąd jest w stanie wylecieć tylko w jednym miejscu to czy nie lepiej byłoby go zadeklarować w miejscu gdzie jest rzucany? W imię wysokiej kohezji;D

Reply:
Miałem taką myśl, ale miałem już jeden zdefiniowany, to dodałem, żeby nie było mu przykro samemu ;)